src/apps/opening-hours-editor/DialogFomular.tsx

```diff
@@ -0,0 +1,70 @@
+// I dont know why eslint is complaining about label-has-associated-control
+// as the label is associated with the input field. I will disable it for now.
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from "react";
+import { EventImpl } from "@fullcalendar/core/internal";
+import { extractTime, updateEventTime } from "./helper";
+
+type DialogFomularProps = {
+  eventInfo: EventImpl;
+  handleEventEditing: (eventInfo: EventImpl) => void;
+};
+
+const DialogFomular: React.FC<DialogFomularProps> = ({
+  eventInfo,
+  handleEventEditing
+}) => {
+  const [title, setTitle] = useState(eventInfo.title);
+  const [startTime, setStartTime] = useState(extractTime(eventInfo.startStr));
+  const [endTime, setEndTime] = useState(extractTime(eventInfo.endStr));
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    eventInfo.setProp("title", title);
+    const startDate = updateEventTime(eventInfo.startStr, startTime);
+    const endDate = updateEventTime(eventInfo.endStr, endTime);
+    eventInfo.setDates(startDate, endDate);
+
+    handleEventEditing(eventInfo);
+  };
+
+  return (
+    <form
+      onSubmit={handleSubmit}
+      style={{ display: "grid", marginTop: "20px" }}
+    >
+      <label htmlFor="title">Title:</label>
+      <input
+        id="title"
```

Comment:
Isn't `title` a too generic id?
How about `dialog-formular-title`.
Same goes for the rest of the ids...